### DISCUSSION TITLE
Feltprefiks for egne prosjektegenskaper

### DISCUSSION THREAD

**Discussion by olemp** [Puzzlepart/prosjektportalen365#438] [Q&A]
Per nå er det kun `Gt` prefiks som er støttet. Dette kan fort føre til at det blir kræsj i feltnavn når man vil legge til egne felter.

Foreslår et ekstra statisk prefiks for egne felter som legges til over Prosjektportalen 365. 

Tanker @tarjeieo @pzljanb?

_Tester ut nye GitHub discussions_

**Comment by pzljanb** [comment]
Jeg stemmer i på den :-)
Vil foreslå nytt Prefix = 'User'

**Reply by olemp** [reply]
> Jeg stemmer i på den :-)
> Vil foreslå nytt Prefix = 'User'

Jeg tenker at User fort kan kræsje med brukerfelter i SharePoint. Foreslår noe ala `Shp` (for Sherpa, ring a bell @tarjeieo?) eller `Gtc` (Glittertind custom).

**Reply by olemp** [reply]
Vi kan eventuelt se på om vi kan belage oss på å bruke feltgrupper istedenfor feltprefiks. Det er muligens enda bedre. Da kan kunden bruke hvilke prefiks de vil.

**Reply by pzljanb** [reply]
Tenkte ikke på at User kunne kræsje med andre, men det er et godt poeng.
Da er `Gtc` et veldig bra alternativ
Feltgrupper er muligens det aller beste, om ikke den er for krevende (i timer) å implementere?

**Reply by olemp** [reply]
> Tenkte ikke på at User kunne kræsje med andre, men det er et godt poeng.
> Da er `Gtc` et veldig bra alternativ
> Feltgrupper er muligens det aller beste, om ikke den er for krevende (i timer) å implementere?

Ikke nødvendig. Men går litt tid på å sette seg inn i hvor vi bruker prefiks for å hente felter, og om det kan erstattes av «lookup» på feltgruppe.

**Reply by olemp** [reply]
Tok en kjapp titt. Bør ikke ta mer enn 1-3 timer å fikse dette til å bruke feltgruppe istedenfor. Vi kan da bruke en av gruppene vi allerede bruker for feltene våre (Gt-feltene) eller at vi støtter en egen gruppe for formålet. 

Da sjekker vi om feltet har Gt-prefiks **eller** om feltet er i, for eksempel, gruppen «Kundespesifikke prosjektegenskaper».

**Reply by olemp** [reply]
Tenker vi kjører på med dette og oppdaterer [wiki-siden](https://github.com/Puzzlepart/prosjektportalen365/wiki/Egne-prosjektegenskaper) @pzljanb?

**Reply by pzljanb** [reply]
Helt enig @olemp, og gjør du det på 3 timer har jeg kundeprosjekt til deg også :-)

**Reply by olemp** [reply]
> Helt enig @olemp, og gjør du det på 3 timer har jeg kundeprosjekt til deg også :-)

Send det på privven 🚀

**Reply by olemp** [reply]
Laget issue. Låser diskusjonen etterhvert.

https://github.com/Puzzlepart/prosjektportalen365/issues/439

**Reply by tarjeieo** [reply]
Liker godt forslaget med egen gruppe!
